tests/README.md

```diff
@@ -0,0 +1,114 @@
+* Label the case with ***CI.Day3***
+* Label the case with importance ***CI.Critical*** or ***CI.High***
+* Don't need to run creation step, just in BeforeEach step call function  ***PrepareRHCSClusterByProfileENV()*** it will load the clusterID prepared
+* Code for day3 actions and check step
+* Every case need to recover the cluster after the case run finished unless it's un-recoverable
+
+### Labels
+
+* Label your case with the ***CI.Feature<feature name>*** defined in rosa/tests/ci/labels/features.go
+* Label your case with importance defined in rosa/tests/ci/labels/importance.go
+* Label your case with ***CI.Day1Post/CI.Day2/CI.Day3*** defined in rosa/tests/ci/labels/runtime.go, according to the case runtime
+* Label your case with ***CI.Exclude*** if it fails CI all  the time and you can't fix it in time
+* Label you case with ***CI.NonClassicCluster/CI.NonHCPCluster*** if it does not fit a type of cluster
+
+## Running
+
+The cluster created by the automation scripts are shared across all of the test cases. [Why we need do this?](./docs/challenge.md).
+
+### Prerequisite
+
+Please read repo's [README.md](../README.md)
+For the test cases, we need `make install` to make the terraform provider installed to local
+
+#### Users and Tokens
+
+Please login ocm and aws cli prior to launching the tests.
+
+#### Global variables
+
+<!-- To declare the profile cluster type to be run, use the below variable::
+* export CLUSTER_PROFILE = <rosa-tf-profile> -->
+
+### Running a local CI simulation
+
+This feature allows for running tests through a case filter to simulate CI. Anyone can customize the case label filter to select the specific cases that would be run. 
+
+<!-- * Prepare cluster with profile
+  * Check file rosa/tests/ci/profiles/tf_cluster_profile.yml to find the supported profiles and detailed configuration
+    * `export CLUSTER_PROFILE=<profile name>`
+  * Export the token
+    * `export RHCS_TOKEN=<rhcs token>`
+  * Run ginkgo run command
+    * `ginkgo run --label-filter day1-prepare tests/e2e` -->
+* Run day2 day1 post cases with profile
+  * Check file rosa/tests/ci/profiles/tf_cluster_profile.yml to find the supported profiles and detailed configuration
+    * `export CLUSTER_PROFILE=<profile name>` #If it had been run, then you can skip
+  * Run ginkgo run command
+    * `ginkgo run --label-filter '(Critical,High)&&(day1-post,day2)&&!Exclude' tests/e2e`
+* Run a specified case to debug
+  * `ginkgo -focus <case id> tests/e2e`
+<!-- * Run destroy with profile
+  * Check file rosa/tests/ci/profiles/tf_cluster_profile.yml to find the supported profiles and detailed configuration
+    * `export CLUSTER_PROFILE=<profile name>` #If it had been run, then you can skip
+  * Export the token
+    * `export RHCS_TOKEN=<rhcs token>` #If it had been run, then you can skip
+  * Run ginkgo run command
+    * `ginkgo run --label-filter destroy tests/e2e` -->
+
+### Set log level
+
+* Log level defined in rosa/tests/utils/log/logger.go
```

Comment:
I didn't find the logger.go file .

Reply:
update in progress

Reply:
I think it would be worth using the same logger, but I would not include it in this change. I would prefer a follow up change to leverage the CLI logger. This is a large enough PR as it is, lets not over complicate it where possible.

Reply:
Created https://issues.redhat.com/browse/OCM-6706